Task10_Auth/src/main/java/Mapper.java

```diff
@@ -20,4 +23,29 @@
             throw new RuntimeException("Can't map to the object of " + clazz.getName());
         }
     }
+
+    public static String UserDtoToString(List<UsersDto> users) {
+        ObjectMapper objectMapper = new ObjectMapper();
+        String usersString = null;
+        try {
+            usersString = objectMapper.writeValueAsString(users);
+
+        } catch (JsonProcessingException e) {
+            e.printStackTrace();
+        }
+        return usersString;
+    }
+
+    public static String UserDtoToString(UsersDto users) {
+        ObjectMapper objectMapper = new ObjectMapper();
+        String usersString = null;
+        try {
+            usersString = objectMapper.writeValueAsString(users);
+
+        } catch (JsonProcessingException e) {
+            e.printStackTrace();
+        }
+        return usersString;
+    }
```

Comment:
I'm surprised, because it is really bad code. Do you really don't know how to make universal method?

Reply:
When I found the answer I've remembered that I've already done it during this course. 

I hope I will never forget that now)

Resolved.